tensorflow/tools/dist_test/python/mnist_replica.py

```diff
@@ -69,7 +69,7 @@
                      "Number of (global) training steps to perform")
 flags.DEFINE_integer("batch_size", 100, "Training batch size")
 flags.DEFINE_float("learning_rate", 0.01, "Learning rate")
-flags.DEFINE_boolean("sync_replicas", False,
+flags.DEFINE_boolean("sync_replicas", True,
```

Comment:
Set the flags arguments default value to `True`, because the program is a distributed training program.